source/data-modeling/associations.txt

```diff
@@ -396,7 +396,7 @@ are embedded in a ``Band`` class. The query returns documents that have a
    :start-after: # start-embedded-query
    :end-before: # end-embedded-query
 
-You can use the ``pluck()`` projection method to retrieve embedded documents
+You can use the ``pluck`` projection method to retrieve embedded documents
```

Comment:
q: a lot of our docs include parentheses with method names; is that not standard for ruby?

Reply:
not standard for ruby